android/capacitor/src/main/java/com/getcapacitor/CapConfig.java

```diff
@@ -135,6 +137,7 @@ private void deserializeConfig(@Nullable Context context) {
         html5mode = JSONUtils.getBoolean(configJSON, "server.html5mode", html5mode);
         serverUrl = JSONUtils.getString(configJSON, "server.url", null);
         hostname = JSONUtils.getString(configJSON, "server.hostname", hostname);
+        path = JSONUtils.getString(configJSON, "server.path", path);
```

Comment:
My understanding is that this was going to be a run-time only config, not supported in the config json?

Reply:
Yeah I don't think it makes sense for now. Removed